modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -837,6 +837,63 @@ struct BaseDefaultFunctor : public BaseFunctor
     static const char* const ocl_kernel_name;
 };
 
+struct GeluFunctor : public BaseDefaultFunctor<GeluFunctor>
+{
+    typedef GeluLayer Layer;
+
+    explicit GeluFunctor() {}
+
+    bool supportBackend(int backendId, int)
+    {
+        return backendId == DNN_BACKEND_OPENCV;
+    }
+
+    inline float calculate(float x) const
+    {
+        return 0.5f * x * (1.0f + erf(x * M_SQRT1_2));
+    }
+
+    int64 getFLOPSPerElement() const { return 1; }
```

Comment:
> 1

4 + erf calculation/approximation.
30-100 is more applicable here.

Reply:
Ok